### DISCUSSION TITLE
[Question] How to pack flet app for android apk

### DISCUSSION THREAD

**Discussion by stabldev** [flet-dev/flet#1705] [Q&A]
### Question

I made a flet app and now I want to pack and run this on android as an apk.
I did `flet pack ...` but its not generating .apk file

Please explain how to do this... I couldnt find a perfect doc regarding this.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Currently, `flet pack` is for packaging desktop apps only. Android and iOS are coming, but the process will be more "involved" and require Flutter SDK.

**Reply by baseplate-admin** [reply]
Hi is there an ETA for this?
